db/models/story.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Story = sequelize.define('Story', {
+    draft: DataTypes.TEXT,
+    published: DataTypes.TEXT,
+    title: DataTypes.STRING,
+    subtitle: DataTypes.STRING,
+    imageLink: DataTypes.STRING,
+    userId: DataTypes.INTEGER,
+    publishAfter: DataTypes.DATE
+  }, {});
+  Story.associate = function(models) {
+    belongsTo(models.User, {foreignKey: 'userId'})
```

Comment:
This is missing a **Story.**belongsTo(models.User, {foreignKey: 'userId'}); for the association.

Reply:
You should see the change in the latest push.